adguard-vpn.com/privacy.html.md

```diff
@@ -56,25 +56,31 @@ For the exact data that the payment provider collects about you, please see the
 
-**Sign-in / Account registration**. When signing in to connect the installed app to your account or registering a new account, you need to provide your email address and create a password to initially create a VPN connection. We process this essential data to provide you with the AdGuard VPN service.
-Using our software. When you choose to use our app, you will provide us your data, including Personal Data. This information, which we must process to ensure the full functionality of AdGuard VPN services, depends on the operating system for which the software is designed. For details, please refer to the corresponding privacy notices.
+**Sign-in/Account registration**. When signing in to connect the installed app to your account or registering a new account, you need to provide your email address and create a password to initially create a VPN connection. We process this essential data to provide you with the AdGuard VPN service.
+
+**Using our software**. When you choose to use our app, you will provide us your data, including Personal Data. This information, which we must process to ensure the full functionality of AdGuard VPN services, depends on the operating system for which the software is designed. For details, please refer to the corresponding privacy notices.
 
 - For AdGuard VPN for Windows: <https://adguard-vpn.com/privacy/windows.html>
 - For AdGuard VPN for Android: <https://adguard-vpn.com/privacy/android.html>
 - For AdGuard VPN for Mac: <https://adguard-vpn.com/privacy/mac.html>
 - For AdGuard VPN for iOS: <https://adguard-vpn.com/privacy/ios.html>
 - For AdGuard VPN Browser Extension: <https://adguard-vpn.com/privacy/extension.html>
 
-**Usage information**. When you use our VPN service, we need to make sure that you stay within the agreed-upon limits. Therefore, in the free version, we collect data on your traffic usage. Additionally, we collect information on the number of instances of our software that have an active VPN connection for both the free and paid versions. We receive this information from our VPN servers through which you establish a VPN connection.
+**Usage information**. When you use our VPN service, we need to make sure that you stay within the agreed-upon limits. Therefore, we collect data on your traffic usage. This is done for two reasons: to prevent misuse of our service and to know when a user's subscription has expired. When the subscription expires or is canceled, the free limit is reactivated and without knowing your previous data usage we cannot properly enforce the rules that limit using the free version.
+
+We store only the number of bytes used by the account, it is not associated with locations. Traffic usage data is stored for 90 days.
+
+Additionally, we collect information on the number of instances of our software that have an active VPN connection for both the free and paid versions. We receive this information from our VPN servers through which you establish a VPN connection.
```

Comment:
It makes sense to mention that this is also done for the same reason -- to make sure that the user stays within agreed-upon limits.